app/controllers/likes_controller.rb

```diff
@@ -1,2 +1,13 @@
 class LikesController < ApplicationController
+  def create
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @like = @post.likes.new(author: current_user)
+
+    if @like.save
+      redirect_to user_post_path(@user, @post)
+    else
+      redirect_to new_user_post_like_path(@user, @post), notice: 'Failed to like the post.'
+    end
+  end
```

Comment:
- [ ] Good job so far, but when I try to create a like nothing happens. When I clicked on the `Add like` button nothing happens, kindly fix that and make sure we can add a like 🙃 

https://github.com/Newtayo/blog_app/assets/74501523/6e49faa1-404e-43f6-a569-52df54fbfc39